pkg/storage/mysql/mysql.go

```diff
@@ -219,10 +219,6 @@ func (s *Datastore) Write(
 	ctx, span := startTrace(ctx, "Write")
 	defer span.End()
 
-	if len(deletes)+len(writes) > s.MaxTuplesPerWrite() {
-		return storage.ErrExceededWriteBatchLimit
-	}
-
```

Comment:
FYI this is already in the command: https://github.com/openfga/openfga/blob/move-tests-part-four/pkg/server/commands/write.go#L177-L179